README.md

```diff
@@ -275,3 +275,31 @@ blocks = cld(length(dd),threads)
 
 @cuda threads=threads blocks=blocks kernel!(de, dd)
 ```
+
+## Deleting a row
```

Comment:
```suggestion
## Applying a function on each field array
```

Reply:
I think the example is useful, but it's a bit odd to emphasize `deleteat!` so much (esp. since at some point I'll add this method). Instead it's good to document that with `foreachfield` the user can create optimized method for all these mutating functions very easily.